torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -258,8 +258,8 @@ To find relevant log snippets:
 4. There should be several instances run (as flaky tests are rerun in CI) from which you can study the logs.
 `;
   let fileInfo = `Test file path: \`${test.file}\``;
-  if (test.file !== `${test.invoking_file}.py`) {
-    fileInfo += ` or \`${test.file}\``;
+  if (test.file !== `${test.invoking_file.replaceAll(".", "/")}.py`) {
```

Comment:
I wonder if we could just set `test.invoking_file = test.invoking_file.replaceAll(".", "/")` and just use `test.invoking_file`